telegram/_chatadministratorrights.py

```diff
@@ -69,18 +69,30 @@ class ChatAdministratorRights(TelegramObject):
             messages of other users.
         can_pin_messages (:obj:`bool`, optional): :obj:`True`, if the user is allowed to pin
             messages; groups and supergroups only.
-        can_post_stories (:obj:`bool`, optional): :obj:`True`, if the administrator can post
+        can_post_stories (:obj:`bool`): :obj:`True`, if the administrator can post
             stories to the chat.
 
             .. versionadded:: 20.6
-        can_edit_stories (:obj:`bool`, optional): :obj:`True`, if the administrator can edit
+            .. versionchanged:: NEXT.VERSION
+            As of this version, this argument is now required. As of our Backward Compatibility
```

Comment:
```suggestion
            As of this version, this argument is now required. As of our stability policy
```

(you can link to it if you like to :))